src/sentry/integrations/slack/requests/command.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from urllib.parse import parse_qs
+
+from sentry.integrations.slack.requests.base import SlackRequest, SlackRequestError
+
+logger = logging.getLogger("sentry.integrations.slack")
+
+
+class SlackCommandRequest(SlackRequest):
+    """
+    A Command request sent from Slack.
+
+    Slack sends the command payload as `application/x-www-form-urlencoded`
+    instead of JSON. This is slightly annoying because the values in the key-
+    value pairs are all automatically wrapped in arrays.
+    """
+
+    def _validate_data(self) -> None:
+        try:
+            qs_data = parse_qs(self.request.body.decode("utf-8"), strict_parsing=True)
+        except ValueError:
+            logger.info("slack.webhook.invalid-payload", extra={"todo": "marcos"})
```

Comment:
did you mean to update the extra field?

Reply:
Oops yes I think there's a helper for that.